sbkeys/generate-aws-sbkeys

```diff
@@ -117,6 +132,9 @@ SBKEYS_SCRIPT="$(mktemp)"
 AWS_KMS_PKCS11_CONF="$(mktemp)"
 cleanup() {
   rm -f "${SBKEYS_SCRIPT}" "${AWS_KMS_PKCS11_CONF}"
+  if [ -n "${CA_BUNDLE:-}" ] ; then
+    rm -f "${CA_BUNDLE}"
```

Comment:
While reading the code, I initially thought that we were removing the user inputted CA_BUNDLE which a user running the command with sufficient privileges would cause it to be removed. But I then noticed the set `CA_BUNDLE="${CA_BUNDLE_FILE}"` a bit earlier in execution. This is more of a readability thing, but should we use a different variable name to avoid confusing the CA_BUNDLE_FILE from the input CA_BUNDLE variable?